app/controllers/autosign_controller.rb

```diff
@@ -3,13 +3,16 @@ def index
     @proxy = SmartProxy.authorized(:view_smart_proxies_autosign).find(params[:smart_proxy_id])
     setup_proxy
 
-    autosign = @api.autosign
-    @autosign = autosign.paginate :page => params[:page], :per_page => Setting::General.entries_per_page
+    @autosign = @api.autosign
```

Comment:
@dLobatog we are currently using the legacy version of datatables. 
@lzap right now this uses the default which is 10, and I feel this fits nicely on this page. Do you think this should be changed?